crates/kilt-dip-primitives/src/merkle_proofs/v0/provider_state/mod.rs

```diff
@@ -176,6 +176,7 @@ impl<
 		ProviderHeader: Decode + HeaderT<Hash = OutputOf<RelayHasher>, Number = KiltBlockNumber>,
 	{
 		let provider_head_storage_key = calculate_parachain_head_storage_key(provider_para_id);
+		log::trace!(target: "dip::consumer::ParachainDipDidProofV0", "Calculated storage key for para ID {:#?} = {:#?}", provider_para_id, provider_head_storage_key);
```

Comment:
Would be nice if all logs have a consistent fmt. Some start with Failed to ..., Some have the failed in the middle.

Reply:
This is not an error message, is a trace about the calculated storage key. I made the error messages consistent in https://github.com/KILTprotocol/kilt-node/pull/630/commits/f99015d526e888041d255c3fb61b109a55af3bc4. This requires a re-approval from you.